dags/analyzer_helper/tests/integration/test_discord_extract_raw_members.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from datetime import datetime
+
+from dags.analyzer_helper.discord.discord_extract_raw_members import DiscordExtractRawMembers
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestDiscordExtractRawMembers(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.client = MongoSingleton.get_instance().client
+        cls.platform_id = 'discord_platform'
+        cls.guild_id = 'discord_guild'
+        cls.db = cls.client[cls.guild_id]
+        cls.collection = cls.db['guildmembers']
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.collection.delete_many({})
+        cls.client.close()
+
+    def setUp(self):
+        self.collection.delete_many({})
+
+    def tearDown(self):
+        self.collection.delete_many({})
+
+    def test_extract_recompute_true(self):
+        sample_data = [
```

Comment:
Here's a sample data that you could use from discord. (note: this is a real data for a bot, please configure or change it yourself). you could use python datetime object for the `joinedAt` and remove the `_id` which MongoDB would automatically assign it.

```
{
  "_id": {
    "$oid": "65cbdccc4b10f0bd1eacbae6"
  },
  "discordId": "159985870458322944",
  "username": "MEE6",
  "roles": [
    "1088165451651092635",
    "1046009276432400435"
  ],
  "joinedAt": {
    "$date": "2023-03-22T18:21:00.870Z"
  },
  "avatar": "b50adff099924dd5e6b72d13f77eb9d7",
  "isBot": true,
  "discriminator": "4876",
  "permissions": "559642693856991",
  "deletedAt": null,
  "globalName": null,
  "nickname": null,
}
```